release/cloudbuild_withsecure_signature.yaml

```diff
@@ -21,16 +21,25 @@ steps:
     args:
       - -c
       - >-
-        gcloud storage cp ${_WITHSECURE_DIR}/${_TEST_TAG_NAME}.txt
-        gs://${_FIRMWARE_BUCKET}/${_FIRMWARE_COMPONENT}/$(echo ${_TEST_TAG_NAME} | sed -e "s/^withsecure_v//")/trusted_os_manifest.txt
+        gcloud storage cp ${_WITHSECURE_DIR}/${_TEST_TAG_NAME}
+        gs://${_FIRMWARE_BUCKET}/${_FIRMWARE_COMPONENT}/$(echo ${_TEST_TAG_NAME} | sed -e "s/^withsecure_v//")/trusted_os_manifest
+  ### Delete the manifest that was only signed by Transparency.dev, since the
+  ### manifest above should be signed by both Transparency.dev and WithSecure.
```

Comment:
It would be good to have a step here which runs a tool that verifies:
- both signatures are present and valid
- the message in the note is identical to the one in our signed manifest (in case e.g WS were to accidentally upload an "old" countersigned manifest)

Worth popping in a TODO?

Reply:
I had a passing thought about #1, but good call on #2 as well. Jotted down both :)